code/game/machinery/vending_machines_vr.dm

```diff
@@ -1528,110 +1529,6 @@
 	icon = 'icons/obj/vending_vr.dmi'
 	icon_state = "Cola_Machine"
 
-//Tweaked existing vendors
-/obj/machinery/vending/hydroseeds/New()
-	products += list(/obj/item/seeds/shrinkshroom = 3,/obj/item/seeds/megashroom = 3)
-	..()
-
-/obj/machinery/vending/security/New()
-	products += list(/obj/item/weapon/gun/energy/taser = 8,/obj/item/weapon/gun/energy/stunrevolver = 4,
-					/obj/item/weapon/reagent_containers/spray/pepper = 6,/obj/item/taperoll/police = 6,
-					/obj/item/clothing/glasses/omnihud/sec = 6)
-	..()
-
-/obj/machinery/vending/tool/New()
-	products += list(/obj/item/weapon/reagent_containers/spray/windowsealant = 5)
-	..()
-
-/obj/machinery/vending/engivend/New()
-	products += list(/obj/item/clothing/glasses/omnihud/eng = 6)
-	contraband += list(/obj/item/weapon/rms = 5)
-	..()
-
-/obj/machinery/vending/medical/New()
-	products += list(/obj/item/weapon/storage/box/khcrystal = 4,/obj/item/weapon/backup_implanter = 3,
-					/obj/item/clothing/glasses/omnihud/med = 4, /obj/item/device/glasses_kit = 1,  /obj/item/weapon/storage/quickdraw/syringe_case = 4)
```

Comment:
Um, but, these vendors won't dispense these things anymore... if you do this...?

Reply:
These were duplicates!